core/kazoo_number_manager/src/knm_numbers.erl

```diff
@@ -662,7 +662,7 @@ reconcile_number(T0, Options) ->
     pipe(do_in_wrap(F1, T0), [fun save_phone_numbers/1]).
 
 do_move_not_founds(Nums, Options) ->
-    pipe(new(Options, Nums)
+    pipe(new([{state, ?NUMBER_STATE_IN_SERVICE} | Options], Nums)
```

Comment:
Can you add a test in knm_create_new_number_test of a move of ?TEST_CREATE_NUM with carrier MDN and check that state is indeed in_service & assigned to reseller?